README.md

```diff
@@ -105,10 +105,10 @@ use Jumbojett\OpenIDConnectClient;
 $oidc = new OpenIDConnectClient('https://id.provider.com',
                                 'ClientIDHere',
                                 'ClientSecretHere');
-$oidc->setResponseTypes(array('id_token'));
-$oidc->addScope(array('openid'));
+$oidc->setResponseTypes(['id_token']);
+$oidc->addScope(['openid']);
```

Comment:
Maybe we can remove this line because the `openid` scope is always added.

Reply:
@ricklambrechts, no problem, I've removed that addScope call 👍